hera_cal/smooth_cal.py

```diff
@@ -99,9 +101,11 @@ def freq_filter(gains, wgts, freqs, filter_scale = 10.0, tol=1e-09, window='none
     '''
     sdf = np.median(np.diff(freqs)) / 1e9 #in GHz
     filter_size = (filter_scale / 1e3)**-1 #Puts it in ns
-    filtered, res, info = uvtools.dspec.high_pass_fourier_filter(gains, wgts, filter_size, sdf, tol=tol, window=window, 
+    (dlys, phi) = fft_dly(gains, wgts, df=sdf*1e9, medfilt=False) #delays are in seconds
+    rephasor = np.exp(-2.0j * np.pi * np.outer(dlys, freqs))
```

Comment:
your exponent here and in line 89 of the test differ by a negative sign. Are you using the right convention?

Reply:
In the test, I'm inserting a delay. It can be positive or negative, depending on the convention. Regardless, this undoes whatever is in there (since dlys can be negative or positive). I've checked that the other convention breaks things.